txpool/tx_object.go

```diff
@@ -122,8 +122,18 @@ func (o *txObject) Executable(chain *chain.Chain, state *state.State, headBlock
 }
 
 func sortTxObjsByOverallGasPriceDesc(txObjs []*txObject) {
-	sort.Slice(txObjs, func(i, j int) bool {
-		gp1, gp2 := txObjs[i].priorityGasPrice, txObjs[j].priorityGasPrice
-		return gp1.Cmp(gp2) >= 0
+	slices.SortFunc(txObjs, func(a, b *txObject) int {
+		switch {
+		case a.priorityGasPrice.Cmp(b.priorityGasPrice) < 0:
+			return 1
+		case a.priorityGasPrice.Cmp(b.priorityGasPrice) > 0:
+			return -1
+		default:
+			if a.timeAdded < b.timeAdded {
+				return 1
+			} else {
+				return -1
+			}
```

Comment:
[minor nit](https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else)
```suggestion
			if a.timeAdded < b.timeAdded {
				return 1
			}
			return -1
```

Reply:
Fixed